godtools/App/Flows/ToolsFlow.swift

```diff
@@ -1,384 +0,0 @@
-//
-//  ToolsFlow.swift
-//  godtools
-//
-//  Created by Levi Eggert on 5/26/20.
-//  Copyright © 2020 Cru. All rights reserved.
-//
-
-import UIKit
-
-class ToolsFlow: ToolNavigationFlow, Flow {
```

Comment:
did you mean to delete this and not move it?

Reply:
Hey Daniel I should have mentioned that in the PR description, but yes I did mean to remove that file.  That logic there was merged some time ago into the App Flow and is no longer being used.